roles/backup/molecule/redhat/prepare.yml

```diff
@@ -2,9 +2,7 @@
 - name: Prepare
   hosts: all
   gather_facts: true
-  pre_tasks:
+  tasks:
     - name: Install crontabs
       ansible.builtin.package:
         name: crontabs
-  roles:
-    - backup
```

Comment:
Do you not still need this backup role to run?

Reply:
not in "prepare", it's still there in "converge":
https://github.com/theforeman/foreman-operations-collection/blob/466fc476b0baa9b45a3efaaa484750215d7482a7/roles/backup/molecule/redhat/converge.yml#L6

it was actually a mistake that it was mentioned here.

Reply:
makes sense